taipy/gui/viselements.json

```diff
@@ -1116,6 +1116,11 @@
                     "shared"
                 ],
                 "properties": [
+                    {
+                        "name": "title",
+                        "type": "str",
+                        "doc": "The title of the metric."
```

Comment:
Missing `default_value`, set to "None"
And this property must appear after "type"

Reply:
resolved